extensions/debuggee/src/org/exist/debugger/client/DebugClient.java

```diff
@@ -21,8 +21,9 @@
  */
 package org.exist.debugger.client;
 
-import jline.ConsoleReader;
-import jline.Terminal;
+//TODO:check where that class would be
+//import jline.ConsoleReader;
+//import jline.Terminal;
```

Comment:
As per previous comment...

Reply:
I did not find where those imported classes come from (trying to get eclipse to build all sources)